deployment/ansible/group_vars/all

```diff
@@ -0,0 +1,7 @@
+java_version: "7u131*"
+terraform_version: "0.9.3"
+awscli_version: "1.11.*"
+aws_profile: "geotrellis-site"
+docker_version: "1.10.*"
+docker_compose_version: "1.9.*"
+ansible_user: "vagrant"
```

Comment:
Ah, Ansible isn't setting this variable because `ansible_local` is being used to provision vs. `ansible`.

In a case like this, it is probably fine to just hard code `vagrant` as the user that gets added to the `docker` group, since development is the only use case Ansible is meant to support.